README.md

```
@@ -2,191 +2,126 @@
 ```bash
-# Initialize database.
-doccano init
-# Create a super user.
-doccano createuser --username admin --password pass
-# Start a web server.
-doccano webserver --port 8000
+docker compose -f docker/docker-compose.prod.yml --env-file .env up
 ```
 
-In another terminal, run the command:
+起動後、ブラウザでhttp://127.0.0.1/にアクセスしてください。
```

Markdownlint (0.35.0)

Bare URL used

(MD034, no-bare-urls)